app/controllers/api/v2/hosts_controller.rb

```diff
@@ -203,6 +203,21 @@ def facts
         render_message(e.to_s, :status => :unprocessable_entity)
       end
 
+      api :POST, "/hosts/:id/rebuild_config", N_("Rebuild TFTP, DNS DHCP configuration")
```

Comment:
a) Comment for throughout really, we may want to add a word into the name describe what sort of config, perhaps something like "rebuild provisioning config", "rebuild orchestrated config" or similar?
b) "TFTP, DNS and DHCP"